src/components/search/searchfields.scss

```diff
@@ -9,3 +9,34 @@
     font-size: 2em;
     align-self: flex-end;
 }
+
+.sorry-text {
+    font-size: 2em;
+    text-align: center;
+    font-family: inherit;
+    width: 100%;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+// New styles for loading circle
+.loading-circle {
```

Comment:
Loading styles should no longer be needed?

Reply:
Fixed.